### ISSUE TITLE
usr/src/app/Gemfile.lock has packages that contain critical security vulnerabilities 

### ISSUE THREAD

**Issue by shaqb** [sitespeedio/sitespeed.io#3602]
### Have you read the documentation?

- [X] Yes, I've read the [how to make a reproducable bug guide](https://www.sitespeed.io/documentation/sitespeed.io/bug-report/)
- [X] Yes, I've read the [how to debug my script guide](https://www.sitespeed.io/documentation/sitespeed.io/scripting/#debug)

### URL

n/a

### What are you trying to accomplish

As part of our organisation we run Trivy scans on public docker images before we can use them internally.

In doing so I can see that the usr/src/app/Gemfile.lock has some packages that contain vulnerabilities.

```
14:07:52  usr/src/app/Gemfile.lock
14:07:52  ========================
14:07:52  Total: 3 (HIGH: 1, CRITICAL: 2)
14:07:52  
+-------------+------------------+----------+-------------------+---------------+---------------------------------------+
|   LIBRARY   | VULNERABILITY ID | SEVERITY | INSTALLED VERSION | FIXED VERSION |                 TITLE                 |
+-------------+------------------+----------+-------------------+---------------+---------------------------------------+
| addressable | CVE-2021-32740   | HIGH     | 2.6.0             | 2.8.0         | rubygem-addressable:                  |
|             |                  |          |                   |               | ReDoS in templates                    |
|             |                  |          |                   |               | -->avd.aquasec.com/nvd/cve-2021-32740 |
+-------------+------------------+----------+-------------------+---------------+---------------------------------------+
| kramdown    | CVE-2020-14001   | CRITICAL | 1.17.0            | 2.3.0         | rubygem-kramdown: processing template |
|             |                  |          |                   |               | options inside documents allows       |
|             |                  |          |                   |               | unintended read access or embedded... |
|             |                  |          |                   |               | -->avd.aquasec.com/nvd/cve-2020-14001 |
+             +------------------+          +                   +---------------+---------------------------------------+
|             | CVE-2021-28834   |          |                   | 2.3.1         | rubygem-kramdown: allows arbitrary    |
|             |                  |          |                   |               | classes to be instantiated            |
|             |                  |          |                   |               | -->avd.aquasec.com/nvd/cve-2021-28834 |
+-------------+------------------+----------+-------------------+---------------+---------------------------------------+
```

Is it possible to update addressable to 2.8 and kramdown to 2.3.1?

### What browser did you use?

Other

### How to reproduce

```shell
Run trivy scan
```

**Comment by soulgalore** [comment]
This should be fixed in 23.5.2.

**Comment by shaqb** [comment]
Thanks so much @soulgalore !
